src/parser/jobs/pld/index.tsx

```diff
@@ -19,8 +19,8 @@ export const PALADIN = new Meta({
 	</>,
 
 	supportedPatches: {
-		from: '✖',
-		to: '✖',
+		from: '7.0',
+		to: '7.01',
 	},
 
 	contributors: [
```

Comment:
I would add yourself as a Developer contributor so you get credit!

Reply:
Done.

Reply:
Or maintainer; we've historically preferred to have at least one person marked with `ROLES.MAINTAINER` before we flag a job as supported, and that person (or persons) has the responsibility of being the lead point of contact for any bug reports/feature requests that come up (not to say other folks aren't allowed to address them, but still). It's not super important (and we're actually discussing whether to drop the maintainer role entirely), but I wanted to mention it all the same.

Reply:
@Toastdeib I was the maintainer from last expansion and had planned on doing the update this weekend, but @czyba beat me to it! That being said, if they are more active, I don't mind handing the maintainer role off to them if they would prefer to have it, but but otherwise, I can add myself.

Reply:
I do not believe I'm cut out for the role. For one, I'm mostly inactive on discord, unless someone actively pings me. Additionally, I'm not familiar with webdev, nor do I have any aspirations to be. You also mentioned that the removal of the maintainer role is also discussed, so...

Though I do wonder, given this PR now has 5 people commenting on it, how to actually go about getting it approved?Especially if there are no maintainers who can have the final say? Do I have to convince everyone who commented on the PR? Are there some general maintainers who can accept a PR, even if they are not particularly interested in the job in question?

Reply:
For our PRs as a whole, 1 repo level maintainer (which includes @Toastdeib and myself) will have to approve the PR in order for it to merge.  Generally the first person who picks it up for review will work with you to see it through to merging.  

I only jumped in with comments on this one because I saw a couple comments that were incorrect that I wanted to help get you the correct information for before you wasted time doing the wrong thing.  Usually, I'd let Toast keep working with you on this unless he stopped actively reviewing it.

We do also encourage job maintainers (of which @Styrfire was the PLD job maintainer last expansion) to help comment on PRs from the "this is accurate for job analysis" perspective, because sometimes our repo maintainers aren't up-to-speed on a given job.

Reply:
Ahhh, got it. Thanks for the explanation.